Week1/homework/js-exercises/remainder.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+//1. If x equals 7, and the only other statement is x = x % 3, 
+//what would be the value of x after the calculation?
+
+let x = 7;
+x = x % 3;
+console.log(x);
+/* When we divide 7 by 3, the remainder will be 1 and the new value for x 
+will be equal 1. So, 1 will be logged.*/
+
+//2. If y equals 21, and the only other statement is y = y % 4, 
+//what would be the value of y after the calculation?
+
+let y = 21  ;
+y = y % 4;
+console.log(y);
+/* When we divide 21 by 4, the remainder will be 1 and the new value for y 
+will be equal 1. So, 1 will be logged.*/
+
+//3. If z equals 13, and the only other statement is z = z % 2, 
+//what would be the value of z after the calculation?
+
+let z = 13  ;
+z = z % 2;
+console.log(y);
+/* When we divide 13 by 4, the remainder will be 1 and the new value for z 
+will be equal 1. So, 1 will be logged.*/
```

Comment:
looks good to me, better to use new variables for remainders instead of assigning to the same ones